Lesson_13/homework_13.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
```

Comment:
```suggestion
import pandas as pd
```
навіщо лізти туди, що ми не вчили?

Reply:
я крутився як міг, бо не вдавалося. А потім зрозумів, що в ту сторону пішов та просто не видалив імпорт))